yoti-sdk-api/src/main/java/com/yoti/api/client/AgeVerification.java

```diff
@@ -0,0 +1,36 @@
+package com.yoti.api.client;
+
+/**
+ * Wraps an 'Age Verify/Condition' attribute to provide behaviour specific to verifying someone's age
+ */
+public interface AgeVerification {
+
+    /**
+     * The type of age check performed, as specified on dashboard. Currently this might be 'age_over' or 'age_under'
+     *
+     * @return The type of age check performed
+     */
+    String getCheckPerformed();
+
+    /**
+     * The age that was that checked, as specified on dashboard.
+     *
+     * @return The age that was that checked
+     */
+    int getAgeVerified();
```

Comment:
Here I would either put simply `getAge()`, because you call it in the context of an age verification,
or `getVerificationAge()`.